examples/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -138,6 +139,7 @@
           "404": {
             "description": "Returned when the resource does not exist.",
             "schema": {
+              "type": "string",
```

Comment:
These additions make me a bit nervous. Have you checked this is still a valid swagger file? Is it correct to have these here now?

Reply:
I honestly wonder how it was correct to not have the `type` field there. I will admit, I have done very little with swagger but from what I have seen it seems more typical to always have a type field and only sometimes have the format field. 
I was able to paste the generated swagger `a_bit_of_everything.swagger.json` into https://editor.swagger.io/ and it seemed happy with the `type` related stuff. It complained about something unrelated to that though.